### ISSUE TITLE
[BUG]: Hover is not working for Adjustment Angle

### ISSUE THREAD

**Issue by Sleeepy2** [jneilliii/OctoPrint-BedLevelVisualizer#579]
**Describe the bug**
When hovering over a point on the Adjustment angle (degree) it should the rotation amount. Now it doesn't show anything.

**Expected behavior**
When hovering over a point on the Adjustment angle (degree) it should the rotation amount.

**Screenshots**
![image](https://user-images.githubusercontent.com/1533684/208992019-0a8fa06a-1d43-4a83-9faf-1810156c232c.png)

**Firmware and Version**
OctoPrint 1.87.6
Python 3.9.2
OctoPi 1.0.0 RC3

**Comment by jneilliii** [comment]
this appears to have been a plugin conflict with Tasmota and TPLinkSmartplug plugins bundled ploty libraries. I have released new rc versions on those plugins to resolve the conflict, and hover should work again, but only on the graph part, not the table below. points on table below will be highlighted while hovering the graph. still contemplating removal of the table completely in favor of the hover popup on the mesh.

**Comment by Ltek** [comment]
I dont think this is solved. I do not see a Adjustment Angle on the graph. I dont use/have the two plugins you mention. sI only see Adjustment Angle if I open Settings (gear icon) and look at the raw data table.

![7c881904-df71-4c30-adcc-6ce018184a02](https://user-images.githubusercontent.com/1626802/216643688-c2296cc4-642b-4a1c-8fe4-39abdd1560c9.png)

**Comment by jneilliii** [comment]
@Ltek are you running the rc version? In OctoPrint's Software Update settings change the release channel to `Release Candidate` for the plugin. 

As for your other issue (seems you edited it out so probably found the setting) descending y option resolves that.

**Comment by Ltek** [comment]
I didnt realize we need to turn on all that stuff in settings 👍 
BUT... I'm not sure how accurate that table is, my bed is darn level but table shows major adjustments needed?
![Capture](https://user-images.githubusercontent.com/1626802/216675955-29c7a017-a7e2-41ff-824c-06cdaf5a8dbe.JPG)

**Comment by jneilliii** [comment]
did you have to use relative z offset setting to get the graph to show in the view? I'm assuming you may have, because I think the adjustment angles are based on getting to 0 (I didn't program that bit of the plugin). Looking at the numbers it's close to a full turn on all of them.

**Comment by Ltek** [comment]
Nope. Dont even know what 'relative z offset' means or does. my settings screenshot...
![Capture2](https://user-images.githubusercontent.com/1626802/216700991-cb5aaed1-b33e-4be8-8c2a-09509019518a.JPG)

**Comment by jneilliii** [comment]
Not sure where those high calculations are coming from then, do they make sense relative to the mesh data itself? I've always thought the adjustment calculations didn't make sense unless probe points landed directly on top of you screws.
